.travis.yml

```diff
@@ -1,6 +1,8 @@
 # Forces travis to use VM insted container, required to be able to build containers.
 sudo: required
 
+cache: pip
```

Comment:
It the docs, it mentions it caches `$HOME/.cache/pip`

https://docs.travis-ci.com/user/caching/#pip-cache